json-endpoints/class.wpcom-json-api-comment-endpoint.php

```diff
@@ -81,6 +81,8 @@ function get_comment( $comment_id, $context ) {
 				&&
 					isset( $this->api->token_details['user'] )
 				&&
+					isset( $this->api->token_details['user']['user_email'] )
```

Comment:
seeing this, shouldn't we check for `isset( $this->api->token_details['user']['display_name'] )` too?